modules/generator/processor/servicegraphs/servicegraphs.go

```diff
@@ -178,6 +181,7 @@ func (p *Processor) consume(resourceSpans []*v1_trace.ResourceSpans) (err error)
 						e.ConnectionType = connectionType
 						e.ClientService = svcName
 						e.ClientLatencySec = spanDurationSec(span)
+						e.ClientEndTimeUnixNano = span.EndTimeUnixNano
```

Comment:
should we put this line in the `Span_SPAN_KIND_PRODUCER` section of the switch to show it belongs with that functionality?

Reply:
@joe-elliott 🙃

Reply:
apologies for the delay. after reviewing all of this logic more carefully I think my concern is that we assign values to the Edge struct in the `consume` function and then interpret them in the `onComplete` function. There are a lot of values and not a lot of clarity on how the values flow through to the various metrics. 

in my mind making a change in this function would help with that clarity but reviewing the logic as a whole i now think it's a mistake. we should just faithfully set the values here and let the `onComplete` and `onExpire` functions interpret them as they will. There's nothing in the future that would prevent something unrelated to messaging systems using these values.

I'm good as is. Thank you for your patience and thoughts here!

Reply:
Sorry for the delay here...
So should we resolve this comment? (and the following similar one)